extra_geom/base.py

```diff
@@ -803,6 +803,13 @@ def rotate(self, angles, center=None, modules=np.s_[:], tiles=np.s_[:],
                degrees=True):
         """Rotate part or all of the detector, making a new geometry.
 
+        The rotation is defined by composition of rotations about the axes of
+        the coordinate system (https://en.wikipedia.org/wiki/Euler_angles),
+        i.e. a rotation around the z axis rotates the xy (detector) plan.
+        We use the right-hand rule, xy being the detector plan with z
+        increasing looking toward the detector front plan, x increasing to
+        the left, y increasing to the top. Positive rotations are clockwise.
+
```

Comment:
```suggestion

In other words:

- Positive x angles tilt the top edge of the detector backwards, away from the source
- Positive y angles tilt the right-hand edge (looking along the beam) away from the source
- Positive z angles turn the detector clockwise (looking along the beam)

```

I think some concrete descriptions like this are useful alongside the definitions.